osu.Game/Utils/SentryLogger.cs

```diff
@@ -36,19 +37,17 @@ public class SentryLogger : IDisposable
 
         private readonly OsuGame game;
 
-        public SentryLogger(OsuGame game)
+        public SentryLogger(OsuGame game, Storage? storage = null)
         {
             this.game = game;
 
-            if (!game.IsDeployedBuild || !game.CreateEndpoints().WebsiteRootUrl.EndsWith(@".ppy.sh", StringComparison.Ordinal))
-                return;
-
```

Comment:
I don't suppose removing this was intentional?

Reply:
i don't require review on this, i'm making it for a discussion thread. please just mute this PR.